src/js/views/patients/sidebar/action/action-sidebar-activity-views.js

```diff
@@ -82,8 +91,46 @@ const StateUpdatedTemplate = hbs`
   <div>{{formatMoment date "AT_TIME"}}</div>
 `;
 
+const CommentView = View.extend({
+  className: 'u-margin--b-8 qa-activity-item',
+  template: hbs`
+    <div class="flex">
+      <span class="comment__author-label">{{ initials }}</span>
+      <span class="comment__author-name">{{ name }}</span>
+      <span class="comment__timestamp">{{ formatMoment created_at "AT_TIME" }}</span>
+      {{#if canEdit}}<span class="js-edit comment__icon">{{far "pen"}} {{ @intl.patients.sidebar.action.activityViews.commentView.edit }}</span>{{/if}}
+    </div>
+    <div class="comment__message">{{ message }}{{#if edited_at}}<span class="comment__edited"> {{ @intl.patients.sidebar.action.activityViews.commentView.edited }} </span>{{/if}}</div>
+  `,
+  ui: {
+    edit: '.js-edit',
+  },
+  templateContext() {
+    const clinician = this.model.getClinician();
+    const currentUser = Radio.request('bootstrap', 'currentUser');
+
+    return {
+      canEdit: clinician.id === currentUser.id,
+      name: clinician.get('name'),
+      initials: clinician.getInitials(),
+      edited: this.model.get('edited_at'),
+    };
+  },
+  onRender() {
+    if (!this.model.get('edited_at')) return;
+
+    const template = hbs`{{formatHTMLMessage (intlGet "patients.sidebar.action.activityViews.commentView.editTooltip")  edited = (formatMoment edited "TIME") }}`;
+
+    new Tooltip({
+      messageHtml: renderTemplate(template, this.templateContext()),
```

Comment:
I think this should be 
```
const edited = this.model.get('edited_at');
if (!edited) return;

...
renderTemplate(template, { edited });
```
no need to add this to `templateContext` for the view.